### ISSUE TITLE
cannot export all sub-notebooks/sub-notes in a notebook.

### ISSUE THREAD

**Issue by MatthewLee123** [HappenApps/Quiver#1197]
Hi, when I import a exported notebook, only notes belong to it can be imported. but lost all sub-notebooks and their notes, can you look into this ? also in your new 3.2 version.

thanks.

**Comment by tillt** [comment]
See https://github.com/HappenApps/Quiver/issues/1198

**Comment by MatthewLee123** [comment]
@tillt hi, thanks. but i mean, when you right click your notebook, and select "export notebook". then create a new library and do a import operation, only few notes are back. all sub-notebooks and their notes cannot be imported.

**Comment by tillt** [comment]
@MatthewLee123 ah sorry, got you wrong then - apologies. Deleted the incorrect reference above.

**Comment by ylian** [comment]
@MatthewLee123 That's right. Currently exporting a Quiver notebook as .qvnotebook doesn't export its sub-notebooks. Exporting to other format (Markdown, etc.) does.

I can change the behaviour to export all notes from sub-notebooks. But of course they will all be inside a single notebook. When you import the notebook back, you will get all the notes but not the nested structure.

If you are doing this because you want to use multi-library support, there is a much easier way. Open both libraries, then just drag a notebook from the sidebar into another library's sidebar.

**Comment by MatthewLee123** [comment]
Hi, ylian

thanks the drag and drop meets me well.

Cheers.
